database/seeder/main.go

```
@@ -0,0 +1,191 @@
+            Slug: fmt.Sprintf("%s-tag", name),
+        }
+        db.Create(&t)
+        tags = append(tags, t)
+    }
+    return tags
+}
+
+func linkTagsToPosts(db *gorm.DB, posts []database.Post, tags []database.Tag) {
+    for _, post := range posts {
+        n := rand.Intn(len(tags)) + 1
+        db.Model(&post).Association("Tags").Replace(randomSubset(tags, n))
```

golangci-lint (1.64.8)

Error return value of `(*gorm.io/gorm.Association).Replace` is not checked

(errcheck)